frontend/test/metabase/scenarios/binning/correctness/longitude.cy.spec.js

```diff
@@ -0,0 +1,110 @@
+import { restore, popover, openPeopleTable } from "__support__/e2e/cypress";
+
+describe("scenarios > binning > correctness > longitude", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    openPeopleTable();
+    cy.findByText("Summarize").click();
+  });
+
+  it("Auto bin", () => {
+    openPopoverFromSelectedBinningOption("Longitude", "Auto bin");
+    popover().within(() => {
+      cy.findByText("Auto bin").click();
+    });
+    getTitle("Count by Longitude: Auto binned");
+    cy.get(".bar");
+    cy.get(".y-axis-label")
+      .invoke("text")
+      .should("eq", "Count");
+    cy.get(".x-axis-label")
+      .invoke("text")
+      .should("eq", "Longitude");
```

Comment:
@flamber Is it ok if we assert on the labels just in one test or would you like to have this check in each of the tests here?

Reply:
One time should be plenty. These should not change based on granularity.